src/lib/plot/axis/y-axis.js

```diff
@@ -32,14 +32,21 @@ const propTypes = {
   ])
 };
 
+delete propTypes.axisType;
+
 const defaultProps = {
   orientation: LEFT,
   attr: 'y'
 };
 
 function YAxis(props) {
+  const axisProps = {
+    ...props,
+    axisType: DIRECTION.VERTICAL
+  };
+
   return (
-    <Axis {...props} />
+    <Axis {...axisProps} />
```

Comment:
You can supply additional props here by: `<Axis {...props} axisType={DIRECTION.VERTICAL} />`.
This might be cleaner as it doesn't require the introduction of a new variable (`axisProps` vs `props`).